plugins/talk-plugin-profile-data/client/components/DeleteMyAccount.js

```diff
@@ -63,14 +63,13 @@ class DeleteMyAccount extends React.Component {
         <p className="talk-plugin-auth--delete-my-account-description">
           {t('delete_request.delete_my_account_description')}
         </p>
-        <p className="talk-plugin-auth--delete-my-account-description">
-          {scheduledDeletionDate &&
-            t(
-              'delete_request.already_submitted_request_description',
-              moment(scheduledDeletionDate).format('MMM Do YYYY, h:mm:ss a')
-            )}
-        </p>
         {scheduledDeletionDate ? (
+          <p className="talk-plugin-auth--delete-my-account-description">
```

Comment:
I wasn't able to compile the code, as these two elements need to be wrapped in an enclosing tag (like a `div`) or break each block into their own conditional checks instead.